README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
   },
 );
 ``` 
+<br>
+
+### Apple Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
+
+Pass these values to your app.
+
+2. In your Flutter app, create a `ApplePayComponentConfiguration` instance. It requires a `ApplePayConfiguration` which contains your merchant id and merchant name.
+
+```dart
+final ApplePayConfiguration applePayConfiguration = ApplePayConfiguration(
+  merchantId: MERCHANT_ID,
+  merchantName: MERCHANT_NAME,
+);
+
+final ApplePayComponentConfiguration applePayComponentConfiguration = ApplePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  applePayConfiguration: applePayComponentConfiguration,
+);
+```
+3. Create a `SessionCheckout` by calling the `AdyenCheckout.session.create()` while passing the required properties:
```

Comment:
"while passing the required properties:" 
"and pass the required properties:" 

These are not necesarilly things that would be improtant for a merchant integrating with the beta releases, but when we add these to docs, we will have less differences if the GitHub ones are already relatively close to our style guide.